tests/e2e/br/br.go

```diff
@@ -667,6 +667,72 @@ var _ = ginkgo.Describe("Backup and Restore", func() {
 			framework.ExpectEqual(cleaned, true, "storage should be cleaned")
 		})
 
+		ginkgo.It("start,truncate,stop log backup using old interface", func() {
```

Comment:
It seems this case name is not accurate. We are stoping a scheduled task, right?

Reply:
done